src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -201,6 +204,7 @@ public ExchangeService(long expectedCacheTime,
         this.fpdResolver = Objects.requireNonNull(fpdResolver);
         this.schainResolver = Objects.requireNonNull(schainResolver);
         this.debugResolver = Objects.requireNonNull(debugResolver);
+        this.bidderMediaTypeProcessor = bidderMediaTypeProcessor;
```

Comment:
Then this will become usual: 
`this.bidderMediaTypeProcessor = Objects.requireNonNull(bidderMediaTypeProcessor);`